### ISSUE TITLE
Invisible russian font

### ISSUE THREAD

**Issue by Elvi51** [purogamer/Fluent-for-Steam#19]
After pressing Shift and Tab while in game there is an invisible russian font in the friends tab (circled red in the screenshot) https://drive.google.com/file/d/1oFPLDrEV24QcqN1PtVwuXFtkSfCLJSjP/view?usp=sharing

**Comment by purogamer** [comment]
yes, you're right, unfortunately due to steam limitations you can't round the context menus, but you can adjust the padding and font of that area. I'll keep it in mind for the next big update.

**Comment by purogamer** [comment]
try download and install this font, https://www.cufonfonts.com/font/segoe-ui-variable-static-display

**Comment by Elvi51** [comment]
The problem wasn't about russian font, but it was about the third friend's nickname

**Comment by purogamer** [comment]
That friend of yours was offline or in online status, or ingame.

**Comment by purogamer** [comment]
That friend of yours was offline or in online status, or ingame.

**Comment by Elvi51** [comment]
He was playing the game
